### PR TITLE
fix broken link for cli installation in mac instructions

### PR DESCRIPTION
**Pull request by sjfink**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ You can also get the [source code](https://github.com/openwhisk/openwhisk) and r
 
 For more details about how OpenWhisk works, see [System overview](./about.md).
 
-## Setting up the OpenWhisk CLI
+## Setting up the OpenWhisk CLI 
+<a name="configure-cli" />
```
Comment (markusthoemmes) [inline docs/README.md]: What's this?

File: common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala [hunk 2]
```diff
@@ -65,10 +65,11 @@ class CouchDbRestClient(protocol: String, host: String, port: Int, username: Str
 
     private val poolPromise = Promise[HostConnectionPool]
 
+    private val QUEUE_SIZE = 16 * 1024;
```
Comment (csantanapr) [inline common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala]: this is not related to docs, could you remove?

Comment (sjfink) [inline common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala]: how did that get in there?

**Comment by rabbah** [comment]
@markusthoemmes want to patch and merge? Carlos is away.

**Comment by csantanapr** [comment]
merged 3f14be04835b7727355ec06e801b8e49ad54193f
